README.md

```
@@ -1001,7 +1001,7 @@ dependencies = [
 ]
 ```
 
-2. Create `Containerfile` in top-level directory like following. Update it as needed:
+2. Create a `Containerfile` in your repository root like the following (see `deploy/lightspeed-stack/Containerfile` in this repo for the upstream image definition). Update it as needed:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)